src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  border: 1px solid $white-smoke;
+  border-radius: 5px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  &__img {
+    margin: 32px auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__inform {
+    margin: 0 16px;
+  }
+
+  &__product-name {
+    line-height: 18px;
+    font-weight: 500;
+    font-size: 12px;
+    color: $dark;
+    margin-top: 0;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    color: $gray;
+    font-size: 10px;
+    font-weight: 400;
+    margin-top: 0;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    @include with-centered-content;
+    margin-bottom: 24px;
+  }
+
+  &__title {
+    font-size: 10px;
+    @include margin-color;
+  }
+
+  &__price {
+    @include with-centered-content;
+    margin-bottom: 16px;
+
+    .card__price-title {
+      margin: 0;
+      color: $gray;
+      font-size: 12px;
+    }
+
+    .card__price-amount {
+      @include margin-color;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  .stars {
```

Comment:
The .stars block should be in a separate SCSS file, as it's a separate block from the .card block. Move it to a new file called 'stars.scss'.